fontforgeexe/fontinfo.h

```diff
@@ -0,0 +1,21 @@
+#ifndef FONTFORGE_FONTFORGEEXE_FONTINFO_H
+#define FONTFORGE_FONTFORGEEXE_FONTINFO_H
+
+#include <ggadget.h>
```

Comment:
From the point of view of the executable, `ggadget.` is a library/system header, not a local header, isn't it?